packages/esm-patient-chart-app/src/visit/hooks/useUpcomingAppointments.tsx

```diff
@@ -17,13 +18,14 @@ export interface AppointmentPayload {
   dateHonored?: string;
 }
 
-export function saveAppointment(appointment: AppointmentPayload, abortController: AbortController) {
-  return openmrsFetch(`${restBaseUrl}/appointment`, {
+export const updateAppointmentStatus = async (toStatus: string, appointmentUuid: string) => {
+  const abortController = new AbortController();
+  const statusChangeTime = dayjs(new Date()).format(omrsDateFormat);
```

Comment:
Do you really need to instanatiate and format the date?  Can you not just submit a `new Date()` in the body?

Reply:
I do think you need to format the date into the OMRS date format, but you probably don't need to do the new Date(), ie you could just do dayjs().format(omrsDateFormat), but i might be wrong about that.